remote-config.jsonc

```
@@ -123,16 +120,16 @@
           "message": "View officially-supported add-ons by running `ddev get --list`, or include third-party add-ons with `ddev get --list --all`"
         },
         {
-          "message": "VS Code users: have you tried the DDEV VS Code extension? https://marketplace.visualstudio.com/items?itemName=biati.ddev-manager"
+          "message": "VS Code users love the DDEV VS Code extension? https://marketplace.visualstudio.com/items?itemName=biati.ddev-manager"
```

```suggestion
          "message": "VS Code users love the DDEV Manager extension! https://marketplace.visualstudio.com/items?itemName=biati.ddev-manager"
```
I would call the extension by its actual name instead of talking about a ddev vs code extension. and the sentence is also non question anymore. i would suggest go with the exclamation mark like in the phpstorm plugin tip.